examples/ota-requestor-app/linux/main.cpp

```diff
@@ -85,6 +84,9 @@ static void InitOTARequestor(void)
     gRequestorCore.Init(&(chip::Server::GetInstance()), &gRequestorUser, &gDownloader);
     gRequestorUser.Init(&gRequestorCore, &gImageProcessor);
 
+    // Set the default providers timeout to one minute
```

Comment:
nit: Remove, this comment no longer needed

Reply:
fixed